core/indexer/elasticsearchDatabase_test.go

```diff
@@ -193,7 +210,7 @@ func TestElasticsearch_saveShardValidatorsPubKeys_RequestError(t *testing.T) {
 		},
 	}
 	elasticDatabase := newTestElasticSearchDatabase(dbWriter, arguments)
-	elasticDatabase.SaveShardValidatorsPubKeys(shardId, valPubKeys)
+	elasticDatabase.SaveShardValidatorsPubKeys(shardId, epoch, valPubKeys)
```

Comment:
Could you please add another test with 2 epochs to see that different lists of validatorPubKeys are requested if different epochs are provided?

Reply:
validators list is requested at a upper level in elasticIndexer so there i will add a test with 2 epoch.

Reply:
ok